src/components/SearchInputs.jsx

```diff
@@ -0,0 +1,14 @@
+const SearchInputs = ({ placeholder, Icon }) => {
+  return (
+    <div className="relative flex items-center">
+      <Icon size={20} className="absolute flex left-2 text-gray-400" />
+      <input
+        type="text"
+        className="bg-white rounded-md p-2 md:p-4 w-80 md:w-60 outline-none text-sm pl-7 md:pl-8"
+        placeholder={placeholder}
+      />
+    </div>
+  );
+};
+
+export default SearchInputs;
```

Comment:
This component is named `SearchInputs` but it has one component, please name it `SearchInput` instead